src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -937,6 +937,82 @@ public ContentItemDisplay PostUnPublish(int id)
             }
         }
 
+        /// <summary>
+        /// Send a document to translator
+        /// </summary>
+        /// <param name="sendToTranslate"></param>
+        /// <returns></returns>
+        [EnsureUserPermissionForContent("sendToTranslate.Id", '5')]
+        public async Async.Task<HttpResponseMessage> PostSendToTranslate(SendToTranslate sendToTranslate)
+        {
+            var translator = Services.UserService.GetUserById(sendToTranslate.UserId);
+
+            if (translator == null)
+            {
+                return Request.CreateValidationErrorResponse("The translator could not be found");

```

Comment:
When things are not found generally we want to return a 404, same goes for the other null checks here